rd-net/RdFramework/Tasks/RdFault.cs

```diff
@@ -22,15 +22,17 @@ public RdFault(Exception inner) : base(inner.Message, inner)
       ReasonText = inner.ToString(); //todo Use system capabilities, stack traces, etc
     }
 
-    [SecurityPermission(SecurityAction.Demand, SerializationFormatter = true)]
+    // [SecurityPermission(SecurityAction.Demand, SerializationFormatter = true)]
+    [Obsolete("Obsolete")]
     protected RdFault(SerializationInfo info, StreamingContext context) : base(info, context)
     {
       ReasonTypeFqn = info.GetString(nameof(ReasonTypeFqn));
       ReasonText = info.GetString(nameof(ReasonText));
       ReasonMessage = info.GetString(nameof(ReasonMessage));
     }
 
-    [SecurityPermission(SecurityAction.Demand, SerializationFormatter = true)]
+    // [SecurityPermission(SecurityAction.Demand, SerializationFormatter = true)]
+    [Obsolete("This API supports obsolete formatter-based serialization. It should not be called or extended by application code.", DiagnosticId = "SYSLIB0051", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
A note to not forget to do something about these eventually. I'm not sure we use them at all.